meshpy/simulation_manager/simulation_manager.py

```diff
@@ -122,6 +122,7 @@ def __init__(
         restart_from_prefix="",
         job_name=None,
         feature=None,
+        custom_post_simulation_command=None,
```

Comment:
Yes, so the overall goal is to switch to Queens, I am also in contact with Sebastian regarding that. 
However, in my current workflow I would greatly benefit from having this small flexibility for single run testing.

I have added more documentation and not an error, but a warning, hence the user gets notified, when his command is not executed.

Reply:
When you refer to single run testing, you are referring to a run on the cluster, right? This can somewhat be done at the moment, but I would like to remove this functionality from MeshPy for the previously stated reasons. What will be supported in the future is running a 4C simulation _locally_, i.e., on the process that is running the python MeshPy scrip. If that process happens to be running via _slurm_, you can also run "cluster simulations". However, the setup of the initial _slurm_ job via the bash script will have to be done with some other tool, e.g., Queens.

I am OK with merging this at the moment, but be aware that this will deleted in the near future.